package.json

```diff
@@ -36,7 +32,13 @@
     "vinyl-fs": "^3.0.2",
     "yamljs": "^0.3.0"
   },
-  "optionalDependencies": {},
+  "optionalDependencies": {
```

Comment:
I'm not entirely sure that [optionalDependencies](https://docs.npmjs.com/files/package.json#optionaldependencies) makes sense as a place to put these. Apparently it's typically used for packages that may or may not exist on a particular platform (e.g. a filesystem events polyfill for non-Linux platforms), where the software won't break if that package isn't available.

As is, it would install them by default. Is that behavior that we're trying to avoid? If so, should we just leave these out?

Reply:
I would add them to the `devDependencies` not yhe `optionalDependencies`. We need them to run tests but that's it.